clients/drcachesim/tests/offline-opcode_categories.templatex

```diff
@@ -0,0 +1,32 @@
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Hello world!
+Opcode mix tool results:
+            133 : total executed instructions
+             34 :       mov
+             17 :       mov
+             17 :   syscall
+             16 :       cmp
+             16 :       jnz
+             16 :       sub
+             16 :       lea
+              1 :       and
+
+              4 : sets of categories
+             51 :      move
+             33 :    branch
```

Comment:
(Interesting to see category results: this is going to be a good tool addition.

Hmm, to get 33 it must count `syscall` in addition to `jnz`?  I guess a system call gateway is a kind of branch.  Would everyone call it a branch?  Sorry, I realize this discussion is not really in the scope of this PR: just interesting; not clearly a problem so I guess we leave it and see if someone complains.)

Reply:
Right, https://github.com/DynamoRIO/dynamorio/blob/master/core/ir/x86/decode_table.c#L2467 categorized `syscall` as a branch.
Not really sure if that's a good semantics, but as pointed out, it's outside the scope of this PR.